test/Microsoft.AspNet.Mvc.Core.Test/Rendering/HtmlHelperLabelExtensionsTest.cs

```diff
@@ -250,29 +245,6 @@ public void LabelHelpers_DisplayDisplayNameForProperty_IfNonNull(string displayN
         }
 
         [Fact]
-        public void LabelFor_ConsultsMetadataProviderForMetadataAboutProperty()
-        {
-            // Arrange
-            var modelType = typeof(DefaultTemplatesUtilities.ObjectTemplateModel);
-            var metadataHelper = new MetadataHelper();
-            metadataHelper.MetadataProvider
-                .Setup(p => p.GetMetadataForProperty(It.IsAny<Func<object>>(), modelType, "Property1"))
-                .Returns(metadataHelper.Metadata.Object)
-                .Verifiable();
-
-            var helper = DefaultTemplatesUtilities.GetHtmlHelper(metadataHelper.MetadataProvider.Object);
-
-            // Act
-            var result = helper.LabelFor(m => m.Property1);
-
-            // Assert
-            metadataHelper.MetadataProvider.Verify();
-
-            // LabelFor() falls back to expression name when DisplayName and PropertyName are null.
-            Assert.Equal("<label for=\"Property1\">Property1</label>", result.ToString());
```

Comment:
as discussed offline, this scenario isn't realistic when using `LabelFor()` but the equivalent could certainly happen when using `Label()` in a top-level view.  as long as we still have a `Label()` test with metadata for a `Type`, we're covered.